toolkit/themes/shared/design-system/build.js

```diff
@@ -0,0 +1,40 @@
+const StyleDictionary = require('style-dictionary');
```

Comment:
This might be a personal preference, but I kinda don't like how the node.js API makes you define and maintain separate `build.js` and `config.json` files. It's actually possible to just define one `config.js` file which seems to work just as well and consolidates all the configuration in one place (example [here](https://amzn.github.io/style-dictionary/#/config?id=configuration-file-formats)). What do you think of switching to that? The whole file would look something like this:

```js
// config.js
const StyleDictionary = require('style-dictionary');

StyleDictionary.registerTransform({
  // your transform here
});

StyleDictionary.registerTransformGroup({
  // your transform group here
});

module.exports = StyleDictionary.extend({
  // our config object from config.json goes here
});
```
and that should Just Work ™️ without having to fuss with `buildAllPlatforms` + having separate files